ReleaseProcessAutomation.Tests/IntegrationTests/ReleaseFromDevelopTests.cs

```diff
@@ -423,4 +423,24 @@ public void ReleaseNewBranch_FromDevelop ()
     Assert.That(TestConsole.Output, Does.Not.Contain("Called UpdateAssemblyInfosForDevelopment!"));
     Assert.That(act1, Is.EqualTo(0));
   }
+
+  [Test]
+  public void ReleaseVersion_WithMajorReleaseTagOnMaster_SuggestsAlphaVersions ()
+  {
+
+    ExecuteGitCommand("tag v2.0.0 -m v2.0.0");
+    ExecuteGitCommand("checkout -b develop");
```

Comment:
I'm suprised this just works. Maybe the repro-description was mis-leading?

"repository freshly checked out" means you did just fetch all the data from the server, but you have not actually checked out (i.e. created a local version) of the respective branches.

You cannot test this without using a source repository from which to fetch your local data. If you just have a local repository, all your branches are "checked out" or they would not exist at all.

Reply:
I'm more or less sure that i fixed this at some point during the development process of the new RPA.

Reply:
which is done in the new version...